apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/DoedshendelseJobService.kt

```diff
@@ -19,17 +22,24 @@ class DoedshendelseJobService(
 ) {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
-    fun run() {
+    fun setupKontekstAndRun(context: Context) {
+        Kontekst.set(context)
+        run()
+    }
+
+    private fun run() {
         if (featureToggleService.isEnabled(DoedshendelseFeatureToggle.KanLagreDoedshendelse, false)) {
-            val nyeDoedshendelser = hentAlleNyeDoedsmeldinger()
-            logger.info("Antall nye dødsmeldinger ${nyeDoedshendelser.size}")
+            inTransaction {
```

Comment:
Skulle vi flyttet denne til linje 41, slik at vi ikke ruller tilbake for langt? Dumt å rulle tilbake noe vi har sendt brev på f. eks.

Reply:
Kan det men alt inni her må være i en intransaction nå så da blir det flere av den

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3891/commits/ba2d0df92dff2e907608b8f21de155658cbb9ae8